<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>151</width>
      <height>431</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Capturar Neurona&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Voltaje:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Posici贸n x:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Posici贸n y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_3"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Red:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_4"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Green:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_5"/>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Blue:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_6"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Insertar al inicio</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Insertar al final</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>221</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>305</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Ingrese ID&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Buscar Neurona&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QTableWidget" name="tableWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <row>
        <property name="text">
         <string>ID</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Voltaje</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Posici贸n x</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Posici贸n y</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Red</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Green</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Blue</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Neurona</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Buscar Neurona</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGuardar"/>
   <addaction name="actionRecuperar"/>
   <addaction name="actionBuscar"/>
   <addaction name="actionCapturar_Mostrar"/>
   <addaction name="actionEscenaNeuronas"/>
   <addaction name="actionOrdenarID"/>
   <addaction name="actionOrdenarVoltaje"/>
   <addaction name="actionEscenaGrafo"/>
   <addaction name="actionAplicarFB"/>
   <addaction name="actionGrafoCompleto"/>
  </widget>
  <action name="actionGuardar">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionRecuperar">
   <property name="text">
    <string>Recuperar</string>
   </property>
  </action>
  <action name="actionBuscar">
   <property name="text">
    <string>Buscar</string>
   </property>
  </action>
  <action name="actionCapturar_Mostrar">
   <property name="text">
    <string>Capturar/Mostrar</string>
   </property>
  </action>
  <action name="actionEscenaNeuronas">
   <property name="text">
    <string>EscenaNeuronas</string>
   </property>
  </action>
  <action name="actionOrdenarVoltaje">
   <property name="text">
    <string>OrdenarVoltaje</string>
   </property>
  </action>
  <action name="actionOrdenarID">
   <property name="text">
    <string>OrdenarID</string>
   </property>
  </action>
  <action name="actionEscenaGrafo">
   <property name="text">
    <string>EscenaGrafo</string>
   </property>
  </action>
  <action name="actionAplicarFB">
   <property name="text">
    <string>AplicarFB</string>
   </property>
  </action>
  <action name="actionGrafoCompleto">
   <property name="text">
    <string>GrafoCompleto</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
